docs/tutorials/intro.md

```diff
@@ -183,23 +197,38 @@ If all went well, running the app with `cargo run` should display:
 
 Here's an internationalized date!
 
-*Notice:* By default, `cargo run` builds and runs a `debug` mode of the binary. If you want to evaluate performance, memory or size of this example, use `cargo run --release`. Our example is also using resources in the `json`  format. It is recommended you generate the data in the `postcard` format (and use it with `BlobDataProvider`) for better performance. Gemeratomg data
+*Notice:* By default, `cargo run` builds and runs a `debug` mode of the binary. If you want to evaluate performance, memory or size of this example, use `cargo run --release`. Our example is also using resources in the `json`  format. It is recommended you generate the data in the `postcard` format (and use it with `BlobDataProvider`) for better performance.
+
+# 7. Advanced Data Management
 
 ## Using data from the filesystem
 
-If you have ICU4X data on the file system in a JSON format, it can be loaded via `FsDataProvider`:
+If you have ICU4X data on the file system in a JSON format, it can be loaded via `FsDataProvider`, which needs the `icu_provider_fs` crate.
+
+This provider performs deserialization so is a `BufferProvider`. This means that the feature `"serde"` needs to be enabled on `icu` (or the specific `icu_foo` component crate), and `"deserialize_json"` needs to be enabled on `icu_provider`. There are also `"deserialize_postcard_1"` and `"deserialize_bincode_1"` features available.
+
+The whole thing can be done by running:
+
+
+```sh
+$ cargo add icu --features serde
+$ cargo add icu_provider_fs
+$ cargo add icu_provider --features deserialize_json
+```
+
+You may also run `cargo remove icu_testdata` as we don't need it anymore.
+
+This should generate:
 
 ```toml
 [dependencies]
-icu = "1.0"
-icu_provider_fs = {version = "1.0.0" }
-
-# By default ICU4X doesn't build with any deserialization, but
-# various formats can be enabled with the deserialize_json,
-# deserialize_postcard_1, and deserialize_bincode_1 features on icu_provider
-icu_provider = {version = "1.0.0" , features = ["deserialize_json"]}
+icu = { version = "1.0", features = ["serde"] }
+icu_provider_fs = { version = "1.0.0" }
+icu_provider = { version = "1.0.0" , features = ["deserialize_json"] }
 ```
 
+We can then use it like so:
+
 ```rs
 use icu_provider_fs::FsDataProvider;
 
```

Comment:
This snippet is not runnable, maybe it's too early to show it? We should talk about data generation first

Reply:
It is, we mention having an icu4x clone

Reply:
*If* you have ICU4X data on the file system in a JSON format, it can be loaded via `FsDataProvider`.

I still think `FsDataProvider` should be the last provider anybody uses, so it should not be our first provider example.